cfep-21.md

```diff
@@ -0,0 +1,60 @@
+
+<table>
+<tr><td> Title </td><td> Export license packages for static and header-only libraries </td>
+<tr><td> Status </td><td> Proposed </td></tr>
+<tr><td> Author(s) </td><td> Daniel Ching &lt;carterbox@users.noreply.github.com&gt;</td></tr>
+<tr><td> Created </td><td> Jul 9, 2022</td></tr>
+<tr><td> Updated </td><td> Jul 25, 2022</td></tr>
+<tr><td> Discussion </td><td> NA </td></tr>
+<tr><td> Implementation </td><td> NA </td></tr>
+</table>
+
+## Abstract
+
+This CFEP proposes that header-only and static libraries and should be required
```

Comment:
The second "and" can be removed I think.

Reply:
```suggestion
This CFEP proposes that header-only and static libraries should be required
```